internal/services/network/application_gateway_resource.go

```diff
@@ -3135,14 +3656,17 @@ func expandApplicationGatewayProbes(d *pluginsdk.ResourceData) *[]network.Applic
 				Host:                                utils.String(host),
 				Interval:                            utils.Int32(interval),
 				MinServers:                          utils.Int32(minServers),
-				Path:                                utils.String(probePath),
 				Protocol:                            network.ApplicationGatewayProtocol(protocol),
 				Timeout:                             utils.Int32(timeout),
 				UnhealthyThreshold:                  utils.Int32(unhealthyThreshold),
 				PickHostNameFromBackendHTTPSettings: utils.Bool(pickHostNameFromBackendHTTPSettings),
 			},
 		}
 
+		if len(probePath) > 0 {
```

Comment:
Can we change this to:
```suggestion
		if probePath != "" {
```